### ISSUE TITLE
Can't create new world, stuck at 100%

### ISSUE THREAD

**Issue by officialRubrix** [Iskallia/Vault-public-S1#8]
Hi

Every time I try to make a new VH world it seems to get stuck at 100%

It also seems to warn me about "Experimental Settings" everytime I try to create a world and says they aren't supported, even if I don't change any settings.

I then have to quit the game because it doesn't load (doesn't crash though, was still responding and I was able to close it without the help of Task Manager)

When I go back into VH after quitting the game, I do see the world that I tried to create, but trying to join said world just ends in the game crashing at "Reading World Data"

[2021-03-25-1.log.gz](https://github.com/Iskallia/Vault-public/files/6203093/2021-03-25-1.log.gz)
[latest.log](https://github.com/Iskallia/Vault-public/files/6203094/latest.log)

**Comment by officialRubrix** [comment]
Just wanted to add that I'm using MultiMC, everyone seems to be importing over CurseForge Launcher, maybe that has something to do with it

**Comment by Nimothy** [comment]
Hi @officialRubrix, sorry to hear that you are having issues. 
This is most likely caused by your instance running out of memory. 

Please head over to your settings for MultiMC5 and make sure you increase your Maximum memory allocation to at least: 4096mb. 

![image](https://user-images.githubusercontent.com/6143176/112445781-6add1400-8d47-11eb-95c0-0f9f1a2c845b.png)

**Comment by officialRubrix** [comment]
Hi, I was running the pack on 8GB, is that not enough?

**Comment by scalda** [comment]
Hi you might want to try updating your java version as well as it looks like you are using an old version @officialRubrix

**Comment by officialRubrix** [comment]
Okay, will do. Thanks

**Comment by officialRubrix** [comment]
Hi, I just updated to 1.8.0_281 (latest on Oracle Website) but I'm still getting the issue

**Comment by Enderized** [comment]
I suggest trying to reduce initial Render Distance to max 2 Chunks and try again.
(That's what helped me in the past with other packs)

**Comment by officialRubrix** [comment]
Launching it over CurseForge worked perfectly, I guess its a MultiMC issue?

**Comment by HoogaSwe** [comment]
Can confirm that it is a issue whit MitliMC, had this issue chnaged to forge and it works fine.

**Comment by Nimothy** [comment]
@HoogaSwe @officialRubrix please can you send over your version of MultiMC, this is working fine for me.

**Comment by HoogaSwe** [comment]
![image](https://user-images.githubusercontent.com/81357122/112456106-b4832a00-8d5a-11eb-83a6-0d5bbdcc09ae.png)
![image](https://user-images.githubusercontent.com/81357122/112456267-e3010500-8d5a-11eb-8d38-50f0aac171e4.png)
![image](https://user-images.githubusercontent.com/81357122/112456410-0926a500-8d5b-11eb-8895-0db4f2bbb9ba.png)

**Comment by HoogaSwe** [comment]
I have an logfile, but for security reason can i send you it as a PM?

**Comment by officialRubrix** [comment]
![image](https://user-images.githubusercontent.com/39860503/112458084-b221cf80-8d5c-11eb-9b2a-7d0ceed8995e.png)
my version of MultiMC

**Comment by Nimothy** [comment]
@HoogaSwe, you can send this over to ChufnutII on Discord.

**Comment by HoogaSwe** [comment]
We got to this error when it stops at 100% (we did try run a vanilla instace and that works fine.)
[12:12:25] [Server thread/INFO] [minecraft/ServerLoginNetHandler]: com.mojang.authlib.GameProfile@7d193ff8[id=34dce685-5196-4af3-b179-638fab1bd17b,name=Hooga,properties={textures=[com.mojang.authlib.properties.Property@6e62c138]},legacy=false] (local:E:5ec21d57) lost connection: Internal Exception: java.lang.ClassCastException: net.minecraft.network.login.ServerLoginNetHandler cannot be cast to net.minecraft.client.network.play.IClientPlayNetHandler

**Comment by webolljr** [comment]
The game locks up as soon as it tries to launch a new word. I am using gdlauncher with 8 gigs of ram allocated to the game.

It seems to do fine until it tries to verify you game with mojang and then it just freezes with the 100% screen of the world creation

Here is the end of the gdlauncher log

[12:53:18] [Thread-1/INFO] [wi.en.ModEngineersDecor/]: Config dropper: item-insertion:false
actions.js:2035 [12:53:18] [Thread-1/INFO] [wi.en.ModEngineersDecor/]: Opt-outs:sign_decor
actions.js:2035 [12:53:18] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 31%
actions.js:2035 [12:53:18] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 32%
actions.js:2035 [12:53:19] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 38%
actions.js:2035 [12:53:19] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 43%
actions.js:2035 [12:53:20] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 49%
actions.js:2035 [12:53:20] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 53%
actions.js:2035 [12:53:21] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 58%
actions.js:2035 [12:53:21] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 62%
actions.js:2035 [12:53:22] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 68%
actions.js:2035 [12:53:22] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 72%
actions.js:2035 [12:53:23] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 78%
actions.js:2035 [12:53:23] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 83%
actions.js:2035 [12:53:24] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 86%
actions.js:2035 [12:53:24] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 90%
actions.js:2035 [12:53:25] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Preparing spawn area: 96%
actions.js:2035 [12:53:25] [Render thread/INFO] [minecraft/LoggingChunkStatusListener]: Time elapsed: 12725 ms
actions.js:2035 [12:53:25] [Server thread/INFO] [co.po.ne.NeonCraft/]: HELLO from server starting
actions.js:2035 [12:53:25] [Server thread/INFO] [is.is.co.IskallUtilities/]: Server is starting
actions.js:2035 [12:53:25] [Server thread/INFO] [co.by.BYG/]: BYG: "Server Starting" Event Complete!
actions.js:2035 [12:53:25] [Server thread/INFO] [FTB Utilities Backups/]: Backups folder - /Users/willie/Library/Application Support/gdlauncher_next/instances/Vault Hunters Official Modpack/backups
actions.js:2035 [12:53:25] [Server thread/INFO] [minecraft/IntegratedServer]: Changing view distance to 11, from 10
actions.js:2035 [12:53:28] [Server thread/INFO] [minecraft/ServerLoginNetHandler]: com.mojang.authlib.GameProfile@4d58ce7d[id=ffd3e739-d6d4-4081-bded-1933a76fd36c,name=Ocsisor,properties={textures=[com.mojang.authlib.properties.Property@295679cb]},legacy=false] (local:E:6787b6a6) lost connection: Internal Exception: java.lang.ClassCastException: net.minecraft.network.login.ServerLoginNetHandler cannot be cast to net.minecraft.client.network.play.IClientPlayNetHandler
﻿
​

**Comment by milist1** [comment]
> Launching it over CurseForge worked perfectly, I guess its a MultiMC issue?

How do I do this?
I know the world is loaded because it said villager suffocated but I can't play in it, stuck at 100%.
My log says:

[13:57:23] [Server thread/INFO] [minecraft/ServerLoginNetHandler]: com.mojang.authlib.GameProfile@7bf0e19f[id=5488133a-5a37-4e2a-a4c9-b04b6b5811c5,name=milist_,properties={textures=[com.mojang.authlib.properties.Property@22955784]},legacy=false] (local:E:48e3bab6) lost connection: Internal Exception: java.lang.ClassCastException: net.minecraft.network.login.ServerLoginNetHandler cannot be cast to net.minecraft.client.network.play.IClientPlayNetHandler
[13:57:29] [Server thread/INFO] [minecraft/Entity]: Villager VillagerEntity['Villager'/65, l='ServerLevel[New World]', x=18.28, y=63.56, z=-339.63] died, message: 'Villager suffocated in a wall'

**Comment by Sling-N-Stone** [comment]
I am having the same issue. I have played on 2 VH servers with no issues, but when I try to create a singleplayer world I get stuck at the 100% loading screen. I am using CurseForge and have allocated 8g of RAM and just updated Java today to have the most recent version. I have played other (larger) modpacks thru CurseForge in singleplayer with no issue. Any other ideas of what I might try to get launched?

**Comment by hom3gr0wn** [comment]
Yeah goes to 100% then does nothing have uninstalled updated java i have 16gbs of ram rtx 3070 amd ryzen 7

**Comment by sewHappyMC** [comment]
Hallo. I wanted to check back in to see if anyone had found a fix for this yet? I'm able to play on curse forge but would really like to use multimc 5 for my instance. Has anyone been able to get this pack to work with multimc?
